docs/best_practices_guide.adoc

```diff
@@ -360,7 +360,7 @@ Therefore, we SHOULD use pre_tasks and post_tasks to make it more clear when the
 |===
 
 .Context
-* http://docs.ansible.com/playbooks_tags.html[Ansible doc explaining tags]
+* https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_tags.html[Ansible doc explaining tags]
```

Comment:
We should open a bug and link it here. This should be backported multiple versions.